.gitignore

```
@@ -13,3 +13,4 @@ _asset_bundler_cache
 .jekyll-metadata
 .idea/
 CLIPBOARD.txt
+vendor/bundle
```

If you want, we could make this even broader and just ignore the whole folder. The Jekyll docs [recommend](https://jekyllrb.com/tutorials/using-jekyll-with-bundler/) doing it that way, so I don't anticipate any issues.
```suggestion
vendor/
```